### ISSUE TITLE
Integration with the flowerpot mod

### ISSUE THREAD

**Issue by sbrl** [runsy/cool_trees#19]
The saplings (and trees) provided by this mod are awesome! I'd love to show them off in the [`flowerpot` mod](https://content.minetest.net/packages/sofar/flowerpot/)'s flowerpot, but this mod doesn't yet support it.

It would be great if cool_trees could support integration with the `flowerpot` mod.

Link to the flowerpot mod: <https://content.minetest.net/packages/sofar/flowerpot/>

Link to API instructions: <https://github.com/minetest-mods/flowerpot/blob/master/api.md>

In short, it would look like this:

```lua
if minetest.global_exists("flowerpot") then
	flowerpot.register_node("foo:bar") -- ...where foo:bar is the name of the node for the sapling
end
```

**Comment by runsy** [comment]
Impossible: The trees are too big for pots.

**Comment by sbrl** [comment]
Ah, I wasn't thinking of the trees themselves! Just the saplings :-)

Example screenshot of the aspen sapling from the `default` game: 
![screenshot_20210807_014336](https://user-images.githubusercontent.com/9929737/128582469-0a983578-c534-47cd-9bb0-20c04fab493b.png)

**Comment by runsy** [comment]
Ah OK hahaha. Yes I will implement.
